native_mate/wrappable.cc

```diff
@@ -24,8 +24,11 @@ WrappableBase::~WrappableBase() {
 }
 
 v8::Local<v8::Object> WrappableBase::GetWrapper() {
-  CHECK(!wrapper_.IsEmpty());
-  return v8::Local<v8::Object>::New(isolate_, wrapper_);
+  DCHECK(!wrapper_.IsEmpty());
+  if (!wrapper_.IsEmpty())
+    return v8::Local<v8::Object>::New(isolate_, wrapper_);
+  else
+    return v8::Local<v8::Object>();
```

Comment:
* if we're admitting that wrapper_ can be empty, shouldn't this return a MaybeLocal instead of a Local?

* _(trivial)_ the 'else' is unnecessary

Reply:
Hrm. Seems like we should just do the same thing as Gin: https://cs.chromium.org/chromium/src/gin/wrappable.cc?l=42

```
v8::MaybeLocal<v8::Object> WrappableBase::GetWrapperImpl(v8::Isolate* isolate,
                                                         WrapperInfo* info) {
  if (!wrapper_.IsEmpty()) {
    return v8::MaybeLocal<v8::Object>(
        v8::Local<v8::Object>::New(isolate, wrapper_));
  }
```

Reply:
Yeah returning `MaybeLocal` would be better, but it requires changes on Electron's side, let's do it in another PR.